lib/kino/table.ex

```diff
@@ -1,6 +1,15 @@
 defmodule Kino.Table do
   @moduledoc false
 
+  @type numeric_summary :: %{min: float(), max: float(), mean: float(), nulls: float()}
+
+  @type categorical_summary :: %{
+          top: String.t(),
+          top_freq: integer(),
+          unique: integer(),
+          nulls: float()
+        }
```

Comment:
We can format the summary value into string on the server right? Then we can generalize to (which I believe we already handle on the client):

```elixir
@type summary :: %{String.t() => String.t()}
```

Reply:
I don't get it... 🤔 Do you mean a single string with all the values?

Reply:
Nono, that's just the map typespec. So it would look something like `%{"min" => "1.234", "max" => "4.144", ...}`.

Reply:
Oh! Got it. Yep! We can have strings 🙈

Reply:
Why not `optional(:summary) => %{String.t() => String.t()}` directly?

Reply:
That works, sure :)